utils/node_local_test.py

```diff
@@ -2122,18 +2216,28 @@ def run_in_fg(server, conf):
     while len(pools) < 1:
         pools = make_pool(server)
 
-    dfuse = DFuse(server, conf, pool=pools[0])
+    pool=pools[0]
+        
```

Comment:
(style)  trailing whitespace
(pylint-trailing-whitespace) Trailing whitespace

Reply:
Will fix, but there's a high priority test run in progress currently.